model/application.go

```diff
@@ -13,13 +13,13 @@ type Application struct {
 	// read only: true
 	// required: true
 	// example: 5
-	ID uint `gorm:"primary_key;unique_index;AUTO_INCREMENT" json:"id"`
+	ID uint `gorm:"primary_key;uniqueIndex;AUTO_INCREMENT" json:"id"`
```

Comment:
While at it, could you remove thi uniqueIndex from the primary key columns. Primary key already indicates unique index.